bind/gencffi_func.go

```diff
@@ -109,6 +109,36 @@ func (g *cffiGen) genFuncBody(f Func) {
 
 	g.wrapper.Printf("_cffi_helper.lib.cgo_func_%[1]s(%[2]s)\n", f.id, strings.Join(funcArgs, ", "))
 
+	if f.err {
+		switch nres {
+		case 1:
+			g.wrapper.Printf("if not _cffi_helper.lib._cgopy_ErrorIsNil(cret):\n")
+			g.wrapper.Indent()
+			g.wrapper.Printf("c_err_str = _cffi_helper.lib._cgopy_ErrorString(cret)\n")
```

Comment:
@sbinet 
Here is the reference from CFFI documentation.
>Any operation that would in C return a pointer or array or struct type gives you a fresh cdata >object. Unlike the “original” one, these fresh cdata objects don’t have ownership: they are >merely references to existing memory.

So Go will have an ownership. Hmm, it might problem when Go removes the object before Python referencing this?